forge-gui/res/cardsfolder/upcoming/contaminant_grafter.txt

```diff
@@ -0,0 +1,15 @@
+Name:Contaminant Grafter
+ManaCost:4 G
+Types:Creature Phyrexian Druid
+PT:5/5
+K:Trample
+K:Toxic:1
+T:Mode$ DamageDoneOnce | CombatDamage$ True | ValidSource$ Creature.YouCtrl | ValidTarget$ Player | TriggerZones$ Battlefield | Execute$ TrigProliferate | TriggerDescription$ Whenever one or more creatures you control deal combat damage to one or more players, proliferate.
```

Comment:
the way you translated this trigger it could still fire multiple times when attacking more than one player

Use DamageAll

Reply:
Hmm I don't really understand. DamageAll is to _deal_ damage right. Do you mean DamageDone ?

Reply:
No

you have "one or more" **twice** so the trigger event is the damage event as a whole

Reply:
Ah Didn't know there was a damageall trigger thanks